yotta/lib/cmakegen.py

```diff
@@ -408,11 +408,13 @@ def generateSubDirList(self, builddir, dirname, source_files, component, all_sub
             resource_files = []
             for f in resource_subdirs:
                 for root, dires, files in os.walk(f):
-                    if root.endswith(".xcassets"):
+                    if ".xcassets/" in root or ".bundle/" in root:
+                        pass
```

Comment:
instead of doing this, you can [modify the "dirs" list in place](http://stackoverflow.com/a/19859907/233201), to exclude these directories from being recursed into